packages/jaeger-ui/src/components/TracePage/TraceStatistics/index.tsx

```diff
@@ -33,6 +33,7 @@ type State = {
   sortIndex: number;
   sortAsc: boolean;
   showPopup: boolean;
+  showSorterTooltip: boolean;
```

Comment:
why is it needed to be in the state?

Reply:
previously when I hover on header cell then both tootlips are displayed (our custom tooltip, and antd click tooltip)
So to not display antd click tooltip when I hover on title I used a state.
> Also, I think click tooltip isn't really needed, it's pretty standard behavior to sort by clicking.

So now updated code only shows custom tooltip on hover